Orange/widgets/classify/tests/test_owloadclassifier.py

```diff
@@ -1,13 +1,15 @@
 # Test methods with long descriptive names can omit docstrings
 # pylint: disable=missing-docstring
 import os
-import pickle
 from tempfile import mkstemp
 
 from Orange.classification.majority import ConstantModel
 from Orange.widgets.classify.owloadclassifier import OWLoadClassifier
 from Orange.widgets.tests.base import WidgetTest
 
+import pickle
```

Comment:
A comment here would be nice, to explain why this needs to be after Orange imports and not grouped with python imports at the top.